app/views/hub/clients/show.html.erb

```diff
@@ -294,9 +294,7 @@
           </div>
           <% if @client.intake.is_ctc? && !@client.intake.drop_off? %>
             <div class="client-profile__field-group">
-              <h2 class="text--bold">
-                Prior Year AGI Amount
-              </h2>
+              <h2 class="text--bold">Prior Year AGI Amount</h2>
```

Comment:
unrelated to this ticket but because this was on another line it was adding a space before the text